fhi_users/auth/rest_auth_views.py

```
@@ -341,6 +345,125 @@
+    ):
+        base_product_id, base_price_id = stripe_utils.get_or_create_price(
+            "FP Basic Professional Plan", 2500, recurring=True
+        )
+        metered_product_id, metered_price_id = stripe_utils.get_or_create_price(
+            "Incremental FP Appeal", 1000, recurring=True, metered=True
+        )
+        line_items = [
+            {"price": base_price_id, "quantity": 1},
+            {"price": metered_price_id},
+        ]
+        stripe_recovery_info = StripeRecoveryInfo.objects.create(items=line_items)
```

Ruff (0.8.2)

Undefined name `StripeRecoveryInfo`

(F821)